src/_base/docker/image/console/root/healthcheck.test.sh.twig

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+healthcheck_test()
+{
+    # run any command required which result can be monitored by Docker's HEALTHCHECK
+    {% for step in @('console.healthcheck.test.steps') -%}
+    {{ step|raw }}
+    {% else -%}
+    echo "No healthcheck test steps declared"
+    {% endfor %}
+
+}
+
+healthcheck_test
```

Comment:
we already have https://github.com/inviqa/harness-base-php/blob/1.5.x/src/_base/docker/image/console/root/lib/task/state.sh, could we not just make that templateable instead?

Reply:
State refers to the state of the `application` as an entity (in fact it checks the database in this implementation of `state.sh`)
I think that checking the `health` state of the container is another level then then the state of the application.

Reply:
state refers the the health (state = healthy or not healthy) of the application. There's an arguable case for removing the db check as not really relevant to console for 99% of it's running time

Reply:
exactly my point
`state` -> for test the health of the `application`
Docker `HEALTHCHECK` -> for test the health of the `console` container

with state `console` knows if the app is ok or not
but first we need to know if `console` is ok or not

I agree with you that the dabase connecttion is not a direct indicator of the health of the application, but I think we can address this separately

Reply:
> i.e. console = sleep infinity, not the app
I don't understand the point you make with this comment

Reply:
what's the application you're referring to? `sleep`?

Reply:
webapp container is the application, cron container is the application etc.

Reply:
they have their own checks they would use in their own containers which aren't the docker/image/console/... folder scripts

Reply:
My worry is that we hook the Docker healhcheck to the execution of the `state.sh` script
and in any project, somebody that wants to run the the `ws`'s `state` command to check the health or the app also customise the `state.sh` script to have batter indicators of health.

Doing so it might compromise the health indicator of the container itself.
Keeping them separate avoids confusion
and adds flexibility

that is my rational 

If you tell that this is not a relevant issue
I can remove the `healthcheck.sh.twig` script and alter the `state.sh` to be a twig template and use that in Docker HEALTHCHECK as I think this is what you are suggesting.

Reply:
moved the healhcheck tests into `tast_state' (or `app state`